github/schema.sql

```diff
@@ -52,6 +52,20 @@ CREATE TABLE IF NOT EXISTS public.github_installation_repos (
     PRIMARY KEY (install_id, repo_id)
 );
 
+--
+-- Name: github_installation_repos_soft; Type: TABLE; Schema: public;
+--
+
+CREATE TABLE IF NOT EXISTS public.github_installation_repos_soft (
+    install_id bigint NOT NULL REFERENCES public.github_installations(id) ON DELETE CASCADE,
+    repo_id bigint NOT NULL,
+    repo_node_id text NOT NULL,
+    repo_full_name text NOT NULL,
+    updated_at timestamp without time zone NOT NULL,
+    exists boolean NOT NULL default true,
```

Comment:
```suggestion
    enabled boolean NOT NULL default true,
```
But let's keep the name `exists` in the view for Vadim.

Reply:
you are kidding.
Done.